Testing/ensure keep solving problems

jmlarson1 opened the pull request:
This testing feature ensures that for any problem we currently solve, we don't stop solving it in the future.

To update the 0-1 `solved` matrix, run with `ensure_still_solve_problems = 0`.

jared321 commented:
I was running the POUNDERS tests by
* changing to the `pounders/m/tests` directory
* running MATLAB, and
* executing `runtests` from the prompt.

This asks MATLAB to execute all the code in `Testpounders.m` using the MATLAB unit testing framework.  My two cents, this is a nice and simple way to provide to users a means to fully test POUNDERS.

jmlarson1 commented:
Per a request from @jared321, I have moved all path declarations for MATLAB POUNDERs testing to a single file. This file will error if BenDFO is not in the correct location.

I've also added a README in the POUNDERs directory pointing to how to run these tests.
